src/pages/Login.js

```diff
@@ -0,0 +1,105 @@
+import { useReducer } from "react";
+import { Navigate, useNavigate } from "react-router-dom";
+import { useAuth } from "../context/auth-context";
+import { loginFormReducer } from "../reducers";
+import { login } from "../services";
+import { useDocumentTitle } from "../hooks/useDocumentTitle";
+
+export const Login = () => {
+  useDocumentTitle("Login | Furnishify");
+  const navigate = useNavigate();
+  const [{ email, password, showPass }, formDispatch] = useReducer(
+    loginFormReducer,
+    { email: "", password: "", showPass: false }
+  );
+  const {
+    state: { loading, error, user },
+    dispatch: authDispatch,
+  } = useAuth();
+
+  const handleLogin = async (event) => {
+    event.preventDefault();
+    try {
+      await login({ email, password }, authDispatch);
+    } catch (error) {}
+  };
+
+  if (user) {
+    return <Navigate to="/" />;
+  }
+
+  return (
+    <div className="container">
+      <form className="form" onSubmit={handleLogin}>
+        <h2 className="my-3">Login</h2>
+        <div>
+          <label className="label" htmlFor="email">
+            Email
+          </label>
+          <input
+            className={`input ${error && "invalid"}`}
+            type="email"
+            id="email"
+            value={email}
+            onChange={(e) =>
+              formDispatch({
+                type: "EMAIL",
+                payload: e.target.value.toLowerCase(),
+              })
+            }
+            required
+            placeholder="john.doe@email.com"
+          />
+        </div>
+        <div>
+          <label className="label my-3" htmlFor="password">
+            Password
+          </label>
+          <div className={` ${error && "invalid"} input-icon`}>
+            <input
+              className="input"
+              type={showPass ? "text" : "password"}
+              id="password"
+              value={password}
+              onChange={(e) =>
+                formDispatch({
+                  type: "PASSWORD",
+                  payload: e.target.value.toLowerCase(),
```

Comment:
Is there a particular reason for using toLowerCase() on password? What if user enters a mix of upper and lower case letters?

Reply:
Thanks for pointing this out. I used toLowerCase for email, missed removing it after copying the code.